src/components/LeaderListItem/index.tsx

```diff
@@ -6,12 +6,12 @@ import './style.less';
 import { getRank } from '../../utils';
 
 interface LeaderListItemProps {
-  exp: number
-  image: string
-  name: string
-  placement: number
-  uuid: string
-  rank: number
+  exp: number;
+  image: string;
+  name: string;
+  placement: number;
+  uuid: string;
+  rank: number;
```

Comment:
Similarly to "users", the leaderboard rank type gets repeated a lot. We might want to abstract that as well.